main.sh

```diff
@@ -27,27 +27,52 @@ Response=$(
     --url "$SERVER_URL/api-pub/v1/axe-watcher/gh/$COMMIT_SHA"
 )
 
-IssueCount=$(echo "$Response" | jq .last_run_violation_count)
+LastRunCreatedAt=$(echo "$Response" | jq -r .last_run_created_at)
+LastRunViolationCount=$(echo "$Response" | jq -r .last_run_violation_count)
+LastRunPageStateCount=$(echo "$Response" | jq -r .last_run_page_state_count)
+LastRunNewViolationCount=$(echo "$Response" | jq -r .last_run_new_violation_count)
+LastRunResolvedViolationCount=$(echo "$Response" | jq -r .last_run_resolved_violation_count)
 IssuesOverA11yThreshold=$(echo "$Response" | jq .issues_over_a11y_threshold)
 AxeURL=$(echo "$Response" | jq -r .axe_url)
 ProjectName=$(echo "$Response" | jq -r .project_name)
+DifferenceInPageStateCount=$(echo "$Response" | jq -r .difference_in_page_states)
 
 # Only set outs when running in GitHub Actions.
 if [ -n "${GITHUB_OUTPUT+x}" ]; then
   echo "project=$ProjectName" >>"$GITHUB_OUTPUT"
   echo "axe_url=$SERVER_URL$AxeURL" >>"$GITHUB_OUTPUT"
-  echo "issue_count=$IssueCount" >>"$GITHUB_OUTPUT"
+  echo "created_at=$LastRunCreatedAt" >>"$GITHUB_OUTPUT"
```

Comment:
Should update the `Outputs` section of `README.md` to match

Reply:
Updated